src/screens/SignUp/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useRef } from 'react';
+import { useForm, SubmitHandler } from 'react-hook-form';
+
+import InputText from 'components/Input';
+import { email } from 'utils/inputValidations';
+import { SIGNUP_FIELDS, ERROR_MESSAGES } from 'constants/index';
+import { User } from 'types/types';
+
+import wLogo from '../../assets/wLogo.png';
+
+import styles from './styles.module.scss';
+
+function SignUp() {
+  const { register, errors, handleSubmit, watch } = useForm<User>({
+    mode: 'onChange',
+    reValidateMode: 'onBlur'
+  });
+  const password = useRef({});
+  password.current = watch('password', '');
+  const onSubmit: SubmitHandler<User> = data => {
+    // integrate service
+    // eslint-disable-next-line no-console
+    console.log(data);
+  };
+
+  return (
+    <div className="row full-height center middle">
+      <form
+        className={`column full-height center space-around ${styles.formContainer}`}
+        onSubmit={handleSubmit(onSubmit)}
+      >
+        <img className={styles.formImage} src={wLogo} alt="wolox-logo" />
+        <InputText
+          text={SIGNUP_FIELDS.name}
+          type="text"
+          name="name"
```

Comment:
You can move this inside constants
SIGNUP_FIELDS = {
      firstName: { name: '', placeholder: '', label: '' },
      email: { name: '', placeholder: '', label: '' }
}